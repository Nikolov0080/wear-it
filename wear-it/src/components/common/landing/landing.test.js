import React from 'react';
import renderer from 'react-test-renderer';
import Landing from './landing';
import { BrowserRouter } from 'react-router-dom';

it('renders correctly Landing', () => {
    const tree = renderer
        .create(<BrowserRouter><Landing /></BrowserRouter>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});