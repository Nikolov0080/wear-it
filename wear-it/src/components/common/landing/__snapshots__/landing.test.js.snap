// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Landing 1`] = `
<div
  className="pageInfo"
>
  <div>
    <div>
      <h1>
        Welcome to 
        <i>
          wear-it
        </i>
         
        <br />
         e-commerce 
      </h1>
      <span>
        You can select category from the menu below, but you can
        <span
          className="spanSpecial"
        >
           Add to cart 
        </span>
        and
        <span
          className="spanSpecial"
        >
           order 
        </span>
        products only if you are registered user!
      </span>
      <br />
      <div
        className="authBox"
      >
        <a
          className="linkBtn"
          href="/login"
          onClick={[Function]}
        >
          <div
            className="btn"
          >
            Login
          </div>
        </a>
        <br />
        <a
          className="linkBtn"
          href="/register"
          onClick={[Function]}
        >
          <div
            className="btn"
          >
            Register
          </div>
        </a>
      </div>
    </div>
  </div>
</div>
`;
